decompiler/config/jak2/all-types.gc

```diff
@@ -7421,8 +7421,8 @@
   :flag-assert         #x1300000fb0
   (:methods
     (new (symbol type int) _type_ 0)
-    (add-setting (_type_ process symbol object object object) none 9)
-    (persist-with-delay (_type_ process time-frame object object float object) none 10)
+    (add-setting "Originally called `setting-set` see (anon-function 48 script)" (_type_ process symbol object object object) none 9)
+    (persist-with-delay "Originally called `setting-pers` see (anon-function 46 script)" (_type_ symbol time-frame symbol symbol float int) none 10)
```

Comment:
we should just change the names then. I believe `remove-setting` is also meant to be `unset-setting`.

Reply:
and `set-setting` is meant to be `reset-setting`

Reply:
see:
https://github.com/open-goal/jak-project/blob/ef22563d36e3d7c826e40e7781e9210ebde73c6a/goal_src/jak1/levels/village_common/oracle.gc#L127

Reply:
Are we also going to change jak 1's code then?  It's a pretty large refactor for something that doesn't really add much value -- our name is still decently descriptive (with that said `add` is probably not the best).
I figured it would atleast be interesting to document but not enough to pursue -- for example `setting-pers` is a horribly descriptive name even if it is original.

It's not going to happen in this PR so you should make an issue if it's something you actually want to do.

Reply:
We should do it in Jak 1 as well yes. It's a simple find and replace so it wouldn't be too hard. I really think we should stick to the original names when we can, plus that's half the point of this project really.

If you want to do it later then that's fine.

Reply:
imo we shouldn't stick to the original names when they can be improved for readability sake.

> If you want to do it later then that's fine.
I'm not interested in doing this work at all, I'd much rather spend my time doing something else.